data/schema.js

```diff
@@ -10,6 +10,18 @@ var schema = {
     }
   },
 
+  consumer: {
```

Comment:
You need to increment the schema version at the top of this file for the automatic migrations to happen correctly.